packages/kbn-optimizer/limits.yml

```diff
@@ -130,3 +130,4 @@ pageLoadAssetSize:
   synthetics: 40958
   expressionXY: 33000
   kibanaUsageCollection: 16463
+  profiling: 18628
```

Comment:
Can you explain where this number comes from (or maybe there is a document describing it) ?

Reply:
Sure! It's part of a CI check that keeps an eye on the JavaScript bundle size of plugins. If this limit is exceeded by a certain margin, the PR will fail.